apps/api/src/server/swagger.js

```diff
@@ -89,6 +89,9 @@ export const spec = {
 			documents: [{ guid: '0084b156-006b-48b1-a47f-e7176414db29' }],
 			username: 'test-user@email.com'
 		},
```

Comment:
after changing swagger.js, need to run npm run gen-api-spec in order to update the swagger-output and swagger-types files